runtime/v2/sdk/src/main/java/com/walmartlabs/concord/runtime/v2/sdk/TaskResult.java

```diff
@@ -0,0 +1,97 @@
+package com.walmartlabs.concord.runtime.v2.sdk;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2020 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+import javax.annotation.Nullable;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TaskResult implements Serializable {
+
+    public static TaskResult success() {
+        return new TaskResult(true, null, null);
+    }
+
+    public static TaskResult error(String message) {
+        return new TaskResult(false, message, null);
+    }
+
+    private final boolean ok;
+    private final String error;
+    private Map<String, Object> values;
```

Comment:
As discussed with @brig, we might want to check if the supplied values are `Serializable`.